content/tutorials/_install_ablyrealtime.textile

```diff
@@ -0,0 +1,336 @@
+<%
+  client_id = 'jamie'
+%>
+h2.
+  default: Step 2 - Install Ably
+  swift: Step 2 - Setup an Xcode project and install Ably
```

Comment:
You're assuming we support Swift / Android in our presence demo, which we won't for now